src/language/plain-text.md

```diff
@@ -12,12 +12,22 @@ Plain text does still use tag and string [interpolation](interpolation.html), bu
 
 One common pitfall here is managing whitespace in the rendered HTML. We'll talk about that at the end of this page.
 
+## Free text
+
+Simple free text can be added to the page, without a tag, by using a pipe. Note: Free text like this has various caveats regarding styling, etc.
```

Comment:
We have been calling this "piped text", so this section should be merged with that section that's further down the page.  It may be that:

1. Free text may be a clearer name than "piped text" it's probably closer to what people are looking for.
2. It might be better to have the piped text further up the page.
3. Your description may be clearer than the one we have currently.

I'm happy to hear opinions on this, and I don't have a strong opinion on it of my own. As a _relative_ beginner, you're probably much better placed than I am to suggest what the best wording is for the help files.

Reply:
Yeah see I had no idea that's what Piped Text was intended for. . . it came across as "piped text is a continuation of text in an element", with the existing example being a continuation of `p` element. I tend to agree that my example makes the intended point a bit more clearly.